lib/protocol/keyParser.js

```diff
@@ -595,6 +595,8 @@ OpenSSH_Private.prototype = BaseKey;
     } else {
       ret = [];
     }
+    if (ret instanceof Error)
+      return ret
```

Comment:
```suggestion
      return ret;
```

Reply:
Sorry, I was way to much in TypeScript :D I will fix that.